### PR TITLE
Fix assertion failure introduced by the recent tooltip changes

### PR DESCRIPTION
**Pull request by DakJaniels**

### CODE REVIEW

**Comment by Duckwhale** [comment]
Thanks! Not sure why luacheck doesn't see this assignment. I'll have to look into that when I'm on a different machine.

**Comment by zeptognome** [comment]
the line should be
`
local itemLink = tooltipData.hyperlink
`

Dak added the _

**Comment by Duckwhale** [comment]
Oh, right. I completely missed that - indeed there is only one value now :joy: (Easily fixed, though)

File: Core/GUI/GameTooltipHooks.lua [hunk 1]
```diff
@@ -515,7 +515,7 @@ local function onTooltipSetItem(tooltip, tooltipData)
 		return
 	end
 
-	local _, itemLink = tooltip:GetItem()
+	local _, itemLink = tooltipData.hyperlink
```
Comment (Duckwhale) [inline Core/GUI/GameTooltipHooks.lua:518]:
```suggestion
	local itemLink = tooltipData.hyperlink
```

**Comment by Duckwhale** [comment]
I didn't encounter any issues in my testing, so presumably this should be fine. If not, can revisit after the next release.

**Pull request merged**